src/sdk/models/metadataFilters.ts

```diff
@@ -0,0 +1,86 @@
+export enum MetadataConjunctionOperator {
+	And = 'and',
+	Or = 'or'
+}
+
+export enum MetadataPropertyFilterOperator {
```

Comment:
This doesn't look right and doesn't match the [server-side operators](https://github.com/portalarchitects/skysync-vnext/blob/198997fbe99590622e09930c2ad7572b52f288d7/Connectors/Connectors/src/PortalArchitects.Connectors.Primitives/Metadata/MetadataPropertyFilterOperator.cs#L11).

Reply:
I began with just the server side enum values for interpreting the "op" values on the metadata property filters.  I ended up adding the enum members as values as well to handle interpreting the "function" values on metadata function expressions. I could move the enum member values "Equals, NotEqual, etc" into a different enum?

Reply:
This enum should just be the server side enum values. Anything else should be a separate enum. Also, I don't recall us having function names like 'Equals' and 'NotEquals'. The possible function names comes from the registered function provides, which is defaulted [here](https://github.com/portalarchitects/skysync-vnext/blob/develop/Connectors/Connectors/src/PortalArchitects.Connectors.Transfers/Metadata/MetadataExpressionParser.cs#L28). You could make the default functions into an enum and then change `MetadataFunctionExpression.name` to `MetadataFunctionName | string`.

Reply:
Refactored